src/Microsoft.AspNetCore.SignalR/DefaultHubLifetimeManager.cs

```diff
@@ -124,12 +107,12 @@ public override Task OnDisconnectedAsync(Connection connection)
             return TaskCache.CompletedTask;
         }
 
-        private static async Task WriteAsync(Connection connection, IInvocationAdapter invocationAdapter, InvocationDescriptor invocation)
+        private async Task WriteAsync(Connection connection, HubMessage hubMessage)
         {
             var stream = new MemoryStream();
```

Comment:
`MemoryStream` technically doesn't really need it since all it's resources are managed and will be marked for clean up at the end of the method, but yeah, it's good to maintain the pattern. This code has been changed around anyway though :).

Good catch!